### PR TITLE
Issue 2119: 90th percentile bug

### PR DESCRIPTION
**Pull request by lucasmbrown-usds**
See extensive discussion of what's in this PR here: https://github.com/usds/justice40-tool/issues/2119#issuecomment-1379646207

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/score/constants.py [hunk 1]
```diff
@@ -353,7 +354,7 @@
     field_names.ADJACENT_TRACT_SCORE_ABOVE_DONUT_THRESHOLD: "ADJ_ET",
     field_names.TRACT_PERCENT_NON_NATURAL_FIELD_NAME
     + field_names.PERCENTILE_FIELD_SUFFIX: "IS_PFS",
-    field_names.NON_NATURAL_LOW_INCOME_FIELD_NAME: "IS_ET",
+    field_names.NON_NATURAL_PCTILE_THRESHOLD: "IS_ET",
```
Comment (vim-usds) [inline data/data-pipeline/data_pipeline/etl/score/constants.py:357]: @lucasmbrown-usds 

The frontend is still showing that this is false when the value is above 90.

Screen Shot 2023-01-12 at 11 16 00 AM

**Comment by travis-newby** [comment]
@vim-usds @sampowers-usds Is this PR now OBE, and can it be closed?

**Comment by vim-usds** [comment]
@travis-newby and @sampowers-usds - I think we can remove this PR right?

**Comment by travis-newby** [comment]
I'm good with closing this PR. Part of it was fixed, and the part that wasn't is now captured in #2185.

**Pull request closed without merging**